packages/story-editor/src/components/elementLink/output.js

```diff
@@ -27,18 +27,23 @@ import { withProtocol } from '../../utils/url';
 import { getLinkFromElement } from '.';
 
 function WithLink({ element, children, ...rest }) {
-  const link = getLinkFromElement(element);
-  if (!link?.url?.length) {
+  const link = getLinkFromElement(element) || {};
+  const { url, icon, desc, rel = [] } = link;
+  if (!url) {
     return children;
   }
-  const urlWithProtocol = withProtocol(link.url);
+  rel.push('noreferrer');
+  const relOutput = rel.join(' ').trim();
```

Comment:
Now that you're using `push()` above to add `noreferrer`, no need for the `trim` anymore.

```js
['noreferrer'].join(' ') // 'noreferrer'
['noreferrer', 'foo'].join(' ') // 'noreferrer foo'
[].join(' ') // ''
```